rtd_docs/api.rst

```diff
@@ -286,7 +287,6 @@ results.
     cirq.StepResult
     cirq.Sweep
     cirq.Sweepable
-    cirq.TrialResult
```

Comment:
instead of removing this you should move it to the Deprecated section below: https://github.com/quantumlib/Cirq/pull/3344/files#diff-c69e423203b6c17f4ed493b8ae22952fL742

Reply:
Done.